sdk/src/Core/Amazon.Util/Internal/_bcl/InternalSDKUtils.bcl.cs

```diff
@@ -33,11 +33,8 @@ public static partial class InternalSDKUtils
     {
         private const string MobileServicesFolderName = "AWS Mobile Services";
 
-#if BCL45
-        static string _userAgentBaseName = "aws-sdk-dotnet-45";
-#else
-        static string _userAgentBaseName = "aws-sdk-dotnet-35";
-#endif
+        static string _userAgentBaseName = "aws-sdk-dotnet-462";
```

Comment:
Wasn't sure about this one: Should we use `aws-sdk-dotnet-framework` instead?

Reply:
Yeah, `aws-sdk-dotnet-framework` would probably be better because I don't see us ever needing to support multiple .NET Framework targets.